etc/rust_analyzer_helix.toml

```diff
@@ -0,0 +1,33 @@
+[language-server.rust-analyzer.config.rustc]
+source = "discover"
+
+[language-server.rust-analyzer.config]
+linkedProjects = [
+    "Cargo.toml",
+    "cargo-miri/Cargo.toml",
+    "miri-script/Cargo.toml",
+]
+
+[language-server.rust-analyzer.config.check]
+invocationLocation = "root"
+invocationStrategy = "once"
+overrideCommand = [
+    "env",
+    "MIRI_AUTO_OPS=no",
+    "./miri",
+    "clippy", # make this `check` when working with a locally built rustc
+    "--message-format=json",
+]
+
+# Contrary to what the name suggests, this also affects proc macros.
+[language-server.rust-analyzer.config.buildScripts]
+enable = true
```

Comment:
Why is this needed here? The vscode file doesn't need it.

Reply:
Sorry, a leak from my config I made according to the documentation from `rustc`. The idea behind this setting is running build scripts for more precise code analysis. That's enabled for the `rust` repo.

I can mirror it for VS Code or remove it from here. What do you think?

Reply:
Aren't these enabled by default? AFAIK at least in vscode, they are.

Reply:
According to the docs, yes, but in the settings from `rustc` it is enabled explicitly.

Reply:
Not sure why they are doing that, but no need to copy it then -- let's just rely on the default.